media/js/mozorg/manifesto.js

```diff
@@ -57,16 +59,30 @@ $(function() {
                         .text(window.trans('principle-nav-prev')).appendTo($nav);
 
                     $nav.on('click', 'button', function () {
-                        nav_modal($(this).hasClass('prev') ? -1 : 1);
-                    });
-
-                    window.dataLayer.push({
-                        event: 'manifesto-interaction',
-                        browserAction: 'modal open',
-                        section: section_id.match(/\d+/)[0]
+                        var $this = $(this);
+                        var $section = $('#modal section');
+
+                        // Track arrow clicks on Manifesto quote popup
+                        window.dataLayer.push({
+                            'event': 'manifesto-quote-click-arrow',
+                            'direction': ($this.attr('class') === 'prev') ? 'previous' : 'next',
+                            'quote': $section.attr('data-ga-quote'),
+                            'quoteNumber': $section.attr('data-ga-quote-number')
+                        });
+
+                        nav_modal($this.hasClass('prev') ? -1 : 1);
                     });
                 }
             });
+
+            // Track Manifesto quote clicks
+
+            window.dataLayer.push({
+                'event': 'manifesto-quote-click',
+                'quote': $this.data('ga-quote'),
+                'quoteNumber': $this.attr('data-ga-quote-number')
```

Comment:
Any reason you're using `.attr` here but `.data` above? Would be better to be consistent if possible.